src/site/content/en/fast/browser-level-image-lazy-loading/index.md

```diff
@@ -67,23 +68,31 @@ Today, Chrome already loads images at different priorities depending on where th
 respect to the device viewport. Images below the viewport are loaded with a lower priority, but they're
 still fetched as soon as possible.
 
-In Chrome 76+, you can use the `loading` attribute to completely defer the loading of offscreen images that can be reached by scrolling:
+From Chrome 76+, you can use the `loading` attribute to completely defer the loading of offscreen images that can be reached by scrolling:
```

Comment:
To be honest, now there is good support we can probably lose the previously stuff and just talk about the current way to do it. Or move this stuff to the end, as other options.

Reply:
Tweaked it a bit in https://github.com/GoogleChrome/web.dev/pull/8312/commits/5cb96eda7034bb80df1b7b11315fde17853eb2dc